ocis-pkg/checks/checkgrpc.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // NewGRPCCheck checks the reachability of a grpc server.
-func NewGRPCCheck(address string) func(ctx context.Context) error {
+func NewGRPCCheck(address string) func(context.Context) error {
```

Comment:
I thought we agreed on dropping the context for now, because it has no use atm.

Reply:
It hast, its for cancellation of the err group, see the tests